omnibus/package-scripts/agent/postinst

```diff
@@ -131,9 +131,13 @@ else
     CONF_DIR=$INSTALL_DIR/etc
     RUN_DIR=$INSTALL_DIR/run
 
+    # On Mac, the real log folder is located under the user profile folder, because some MacOS versions deletes
```

Comment:
`under the user profile` is not really the case.

Reply:
I'll change the comment. Thanks for catching that.